src/actions/memberRegistrationActions.js

```diff
@@ -3,109 +3,109 @@ export const checkValidUserInfo = errorMessageObject => ({
- * createStudentSuccessAction action will call when
- * create student AIP response is success
- * @param {Object} newStudent
- * @return {{newStudent: {Object}, type: string}}
+ * createMemberSuccessAction action will call when
+ * create member AIP response is success
+ * @param {Object} newMember
+ * @return {{newMember: {Object}, type: string}}
  */
-export const createStudentSuccessAction = newStudent => ({
-  type: 'CREATE_STUDENT_SUCCESS',
-  newStudent,
+export const createMemberSuccessAction = newMember => ({
+  type: 'CREATE_MEMBER_SUCCESS',
+  newMember,
 });
 /**
- * createStudentFailedAction action will call when
- * create student AIP response is fail
+ * createMemberFailedAction action will call when
+ * create member AIP response is fail
  * @param {String} message
  * @return {{type: string, message: {String}}}
  */
-export const createStudentFailedAction = message => ({
-  type: 'CREATE_STUDENT_FAILED',
+export const createMemberFailedAction = message => ({
+  type: 'CREATE_MEMBER_FAILED',
   message,
 });
 /**
- * fetchStudentDataAction action will call when
- * student data will fetched
+ * fetchMemberDataAction action will call when
+ * member data will fetched
  * @param {String} id
  * @param {String} secretKey
  * @return {{secretKey: *, id: *, type: string}}
  */
-export const fetchStudentDataAction = ({ id, secretKey }) => ({
-  type: 'FETCH_STUDENT',
+export const fetchMemberDataAction = ({ id, secretKey }) => ({
+  type: 'FETCH_MEMBER',
   id,
   secretKey,
 });
 /**
- * fetchStudentSuccessAction action will call when
- * fetch student data AIP response is success
- * @param {Object} student
- * @return {{student: {Object}, type: string}}
+ * fetchMemberSuccessAction action will call when
+ * fetch member data AIP response is success
+ * @param {Object} member
+ * @return {{member: {Object}, type: string}}
  */
-export const fetchStudentSuccessAction = student => ({
-  type: 'FETCH_STUDENT_SUCCESS',
-  student,
+export const fetchMemberSuccessAction = member => ({
+  type: 'FETCH_MEMBER_SUCCESS',
+  member,
 });
 /**
- * fetchStudentFailedAction action will call when
- * fetch student data AIP response is fail
+ * fetchMemberFailedAction action will call when
+ * fetch member data AIP response is fail
  * @param {String} message
  * @return {{type: string, message: String}}
  */
-export const fetchStudentFailedAction = message => ({
-  type: 'FETCH_STUDENT_FAILED',
+export const fetchMemberFailedAction = message => ({
+  type: 'FETCH_MEMBER_FAILED',
   message,
 });
 /**
- * updateStudentDataAction action will call when student data will updated.
+ * updateMemberDataAction action will call when member data will updated.
  * @param {String} id
  * @param {String} secretKey
- * @param {Object} updatedStudent
- * @return {{updatedStudent: {Object}, secretKey: {String}, id: {String}, type: string}}
+ * @param {Object} updatedMember
+ * @return {{updatedMember: {Object}, secretKey: {String}, id: {String}, type: string}}
```

Comment:
Maintain consistency in JS Docs
```suggestion
 * @return {{updatedMember: Object, secretKey: String, id: String, type: String}}
```

Reply:
Done.